homeassistant/components/habitica/button.py

```diff
@@ -74,6 +86,133 @@ class HabitipyButtonEntity(StrEnum):
 )
 
 
+MAGE_SKILLS: tuple[HabiticaButtonEntityDescription, ...] = (
+    HabiticaButtonEntityDescription(
+        key=HabitipyButtonEntity.MPHEAL,
+        translation_key=HabitipyButtonEntity.MPHEAL,
+        press_fn=lambda coordinator: coordinator.api.user.class_.cast["mpheal"].post(),
+        available_fn=lambda data: data.user["stats"]["lvl"] >= 12
+        and data.user["stats"]["mp"] >= 30
+        and data.user["stats"]["class"] == MAGE,
+    ),
+    HabiticaButtonEntityDescription(
+        key=HabitipyButtonEntity.EARTH,
+        translation_key=HabitipyButtonEntity.EARTH,
+        press_fn=lambda coordinator: coordinator.api.user.class_.cast["earth"].post(),
+        available_fn=lambda data: data.user["stats"]["lvl"] >= 13
+        and data.user["stats"]["mp"] >= 35
+        and data.user["stats"]["class"] == MAGE,
+    ),
+    HabiticaButtonEntityDescription(
+        key=HabitipyButtonEntity.FROST,
+        translation_key=HabitipyButtonEntity.FROST,
+        press_fn=lambda coordinator: coordinator.api.user.class_.cast["frost"].post(
+            targetId=coordinator.config_entry.unique_id
+        ),
+        available_fn=lambda data: data.user["stats"]["lvl"] >= 14
+        and data.user["stats"]["mp"] >= 40
+        and data.user["stats"]["class"] == MAGE,
```

Comment:
Please wrap the multiline lambdas to make them stand out from the rest of the parameters